### ISSUE TITLE
property "aid" needs clarification 

### ISSUE THREAD

**Issue by apfister** [koopjs/koop-provider-composite#13]
it isn't clear that `aid` needs to be explicitly included in your JSON schema for your indicator. can we remove this completely and just have `model.js` handle it so the user doesn't need to worry about it? 

https://github.com/koopjs/koop-provider-composite/blob/master/files/put_initiative.sh#L83-L85
&& 
https://github.com/koopjs/koop-provider-composite/blob/master/model.js#L316

**Comment by dmfenton** [comment]
yes I agree with you. users shouldn't have to know anything about this implementation detail
